relay-server/src/utils/unreal.rs

```diff
@@ -199,6 +199,13 @@ fn merge_unreal_context(event: &mut Event, context: Unreal4Context) {
             .set_value(Some(username.into()));
     }
 
+    if let Some(login_id) = &runtime_props.login_id {
+        let user = event.user.get_or_insert_with(User::default);
+        if user.id.0.is_none() {
+            user.id.set_value(Some(login_id.clone().into()));
+        }
+    }
```

Comment:
```suggestion
    if let Some(login_id) = &runtime_props.login_id {
        let id = event.user.get_or_insert_with(User::default).id.value_mut();
        if id.is_none() {
            *id = Some(login_id.clone().into());
        }
    }
```

We can skip the clone/allocation if we take the `login_id` (the same way as above for the `username`) and then using `value_mut` the check for none becomes slightly nicer imo.

Reply:
Let's not take the `login_id` since it removes it from the context and the user may still want to see it in the Sentry UI.